week-1-practise/level-1/9-function-output-2.js

```diff
@@ -6,9 +6,22 @@ var result = s('code', 'your');
 console.log(result);
 
 // Answer these questions:
-// - What does this program do? Run it and see, or Google some of the methods used ot understand them.
+// - What does this program do? Run it and see, or Google some of the methods used ot understand them. 
+//it will join strings and give back a new string with the sum result. In arrays, it will join arrays by order
+//and create new array with the elements of each parameter 
 // - How many parameters does the function take?
+//as many as wanted
 // - What is the function name?
+//s
 // - Where is the function called? with what parameters?
+//line 6, code, your
```

Comment:
The parameters are correct! See previous comments about what is meant by _called_, however.